### ISSUE TITLE
Changes in Node.js TSC charter

### ISSUE THREAD

**Issue by mcollina** [openjs-foundation/cross-project-council#1391]
The Node.js project is asking for a small change in the voting section of its charter: https://github.com/nodejs/TSC/pull/1636.

**Comment by aduh95** [comment]
Would be worth discussing https://github.com/nodejs/TSC/pull/1360 and https://github.com/nodejs/TSC/pull/1650 which are also relatively small changes to the charter.

**Comment by mhdawson** [comment]
With my CPC hat on, I'm +1 on all three of the changes mentioned.

**Comment by UlisesGascon** [comment]
I am +1 to the the three changes with my CPC hat too 👍

**Comment by tobie** [comment]
CPC approved in today's call.

**Comment by aduh95** [comment]
> CPC approved in today's call.

Could you please clarify whether the CPC approved all three PRs, including https://github.com/nodejs/TSC/pull/1360?

**Comment by ovflowd** [comment]
> > CPC approved in today's call.
> 
> Could you please clarify whether the CPC approved all three PRs, including [nodejs/TSC#1360](https://github.com/nodejs/TSC/pull/1360)?

To my understanding, yes.

**Comment by tobie** [comment]
No, we only approved the original one. We entirely missed the other two. Reopening.

**Comment by tobie** [comment]
Notes from this week's CPC call: CPC approved the charter changes we missed last time.
